src/App.jsx

```diff
@@ -14,33 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORTING_OPTIONS = {
+  BY_LENGTH: 'length',
+  ALPHABETICALLY: 'alphabet',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getPreparedGoods = (goods, { sortedField, reversed }) => {
```

Comment:
You can move it in a separate file as helper/util func. Then import and use the inside component. ( for future)